### DISCUSSION TITLE
mini.completion integration

### DISCUSSION THREAD

**Discussion by rxf4e1** [VonHeikemen/lsp-zero.nvim#382] [General]
Hello, i'm just trying to migrate to neovim, and since i like things as minimal as i can do and use more builtins functions as possible, i found on mini.nvim a house, so i'm wondering if lsp-zero has a fine integration with mini.completion, or should i have to configure something else to have it working as it is designed to?

**Comment by VonHeikemen** [comment]
You don't need a special integration. `mini.completion` will work fine with lsp-zero.

Here's the smallest example config I can show.

```lua
require('lazy').setup({
  {'echasnovski/mini.nvim'},
  {'neovim/nvim-lspconfig'},
  {'VonHeikemen/lsp-zero.nvim', branch = 'v3.x'},
})

require('mini.completion').setup({})

local lsp_zero = require('lsp-zero')

lsp_zero.on_attach(function(client, bufnr)
  -- (optional) enable lsp-zero keybindings
  lsp_zero.default_keymaps({buffer = bufnr})
end)

-- example language servers
require('lspconfig').intelephense.setup({})
require('lspconfig').rust_analyzer.setup({})
```

So you can configure lsp-zero like you usually do and everything should work just fine.

**Reply by rxf4e1** [reply]
worked, thanks for your response!
